fw/procfs.c

```diff
@@ -108,52 +108,63 @@ tfw_perfstat_collect(TfwPerfStat *stat)
 static int
 tfw_perfstat_seq_show(struct seq_file *seq, void *off)
 {
-#define SPRNE(m, e)	seq_printf(seq, m": %llu\n", e)
-#define SPRNED(m, e)	seq_printf(seq, m": %ums\n", e)
-#define SPRN(m, c)	seq_printf(seq, m": %llu\n", stat.c)
+#define SPRNE(m, e)  seq_printf(seq, m": %llu\n", e)
+#define SPRNED(m, e)  seq_printf(seq, m": %ums\n", e)
+#define SPRN(m, c)  seq_printf(seq, m": %llu\n", stat.c)
 
-	int i;
+	int i, ret;
 	TfwPerfStat stat = {0};
 	u64 serv_conn_active, serv_conn_sched;
 	SsStat *ss_stat = kmalloc(sizeof(SsStat) * num_online_cpus(),
-				  GFP_KERNEL);
+				GFP_KERNEL);
 	unsigned int val[T_PSZ] = { 0 };
 	TfwPrcntlStats pstats = {.val = val};
 
 	if (!ss_stat)
-		T_WARN("Cannot allocate sync sockets statistics\n");
+    	T_WARN("Cannot allocate sync sockets statistics\n");
```

Comment:
Spaces in identation, tabs must be used. Please check other places in the PR replacing tabs/spaces